Avoid implicit function declarations

For compatibility with future compilers which do not support them
by default.

Include <ctype.h> for toupper, "moveresize.h" for
moveresize_setup_tab, and add a desktops_setup_tab prototype to
"desktops.h" because it is called from main.c.

Submitted upstream: <https://bugzilla.icculus.org/show_bug.cgi?id=6671>

diff --git a/src/appearance.c b/src/appearance.c
index 4fb3f0c12ad9b143..da39273a16a3aea7 100644
--- a/src/appearance.c
+++ b/src/appearance.c
@@ -21,6 +21,8 @@
 #include "tree.h"
 #include "preview_update.h"
 
+#include <ctype.h>
+
 static gboolean mapping = FALSE;
 
 static RrFont *read_font(GtkFontButton *w, const gchar *place, gboolean def);
diff --git a/src/desktops.h b/src/desktops.h
index 1ba3e366d41f5c3d..5f7e32321dbfa3c4 100644
--- a/src/desktops.h
+++ b/src/desktops.h
@@ -24,5 +24,6 @@
 
 void desktops_setup_num(GtkWidget *w);
 void desktops_setup_names(GtkWidget *w);
+void desktops_setup_tab(void);
 
 #endif
diff --git a/src/main.c b/src/main.c
index d7e34469ecc72914..017603537c1f8de1 100644
--- a/src/main.c
+++ b/src/main.c
@@ -28,6 +28,7 @@
 #include "dock.h"
 #include "preview_update.h"
 #include "gettext.h"
+#include "moveresize.h"
 
 #include <gdk/gdkx.h>
 #define SN_API_NOT_YET_FROZEN
